src/Cryptol/REPL/Command.hs

```diff
@@ -2104,6 +2106,19 @@ checkBlock = isolated . go
               subresults <- checkBlock block
               pure (subresult : subresults)
 
+-- | Combine lines ending in a backslash with the next line.
+continuedLines :: [T.Text] -> [T.Text]
+continuedLines xs =
+  case span ("\\" `T.isSuffixOf`) xs of
+    ([], []) -> []
+    (a, []) -> [concat' (map T.init a)] -- permissive
+    (a, b:bs) -> concat' (map T.init a ++ [b]) : continuedLines bs
+  where
+    -- concat that eats leading whitespace between elements
+    concat' [] = T.empty
+    concat' (y:ys) = T.concat (y : map (T.dropWhile isSpace) ys)
```

Comment:
What happens if you have an example like this?

```
/**
 ** ```repl
 ** :check f 1\
 ** 0
 ** ``` */
```

Does this get interpreted as `:check f 10` or `:check f 1 0`?

Reply:
It gets interpreted as 10 which is consistent with C (as an example). This will be useful for long literals, too:

```
x = 0x0123456789abcdef\
      0123456789abcdef\
      0123456789abcdef
```

Reply:
That makes sense—thanks for the explanation. It might be worth testing this property in the `T02` test case.